plugins/sslutils.c

```diff
@@ -201,7 +201,7 @@ int np_net_ssl_check_certificate(X509 *certificate, int days_till_exp_warn, int
 	time_t tm_t;
 
 	if (!certificate) {
-		printf("%s\n", _("CRITICAL - Cannot retrieve server certificate."));
+		printf("%s\n", _("CRITICAL - Not server certificate present to inspect."));
```

Comment:
Would "No server certificate" (on `t` less) be better?

Reply:
good point! Sorry, totally made the commit without reading it over at least once.